app/notes/notes.service.js

```diff
@@ -1,69 +1,75 @@
 (function() {
-  angular.module('meganote.notes')
+  angular
+    .module('meganote.notes')
     .service('NotesService', NotesService);
 
   NotesService.$inject = ['$http'];
   function NotesService($http) {
-    var service = this;
-    service.notes = [];
+    var vm = this;
+    vm.notes = [];
+    vm.create = create;
+    vm.delete = delete(notes);
+    vm.getNotes = getNotes;
+    vm.update = update;
+    vm.find = find;
+    vm.removeById = removeById;
 
-    service.getNotes = function() {
-      var notesPromise = $http.get('http://localhost:3030/');
+
+    function getNotes () {
+      var notesPromise = $http.get('LocalHost');
 
       notesPromise.then(function(res) {
-        service.notes = res.data;
+        vm.notes = res.data;
       });
-
       return notesPromise;
     };
 
-    service.create = function(note) {
-      var notesPromise = $http.post('http://localhost:3030/', {
+    function create(note) {
+      var notesPromise = $http.post('LocalHost', {
         note: note
       });
 
       notesPromise.then(function(res) {
-        service.notes.unshift(res.data.note);
+        vm.notes.unshift(res.data.note);
       });
 
       return notesPromise;
     };
 
-    service.update = function(note) {
-      var notesPromise = $http.put('http://localhost:3030/' + note._id, {
+    function update(note) {
+      var notesPromise = $http.put('LocalHost' + note._id, {
         note: note
       });
 
       notesPromise.then(function(res) {
-        service.removeById(res.data.note._id);
-        service.notes.unshift(res.data.note);
+        vm.removeById(res.data.note._id);
+        vm.notes.unshift(res.data.note);
       });
 
       return notesPromise;
     };
 
-    service.delete = function(note) {
-      var notesPromise = $http.delete('http://localhost:3030/' + note._id);
+    function delete(note) {
```

Comment:
see above. this function name should probably be changed because delete is a reserved keyword in javascript so passing this as a function instead of calling it becomes tricky.